modules/widgets/contact-info.php

```diff
@@ -93,6 +93,8 @@ function widget( $args, $instance ) {
 			 */
 			do_action( 'jetpack_contact_info_widget_start' );
 
+			echo '<div itemscope itemtype="http://schema.org/LocalBusiness">';
```

Comment:
can we use https here?

Reply:
Good question. Looking at `http://schema.org/LocalBusiness`, it lists `Canonical URL` as `http://schema.org/LocalBusiness`. On the other hand, the website works via `https` too.
All the examples on Google's site use `http` and honestly I don't think it matters - this URL is definitely not accessed by any WordPress site using this schema, it's for informational/parsing purposes only AFAIK.

Reply:
Yeah, not a blocker nonetheless.